client/components/price/price-filter.js

```diff
@@ -0,0 +1,76 @@
+angular.module("managerApp").filter("price", [
+    "$translate",
+    function ($translate) {
+        "use strict";
+
+        const frenchTouch = {
+            TTCOnly: false,
+            HTOnly: false,
+            withTTC: true
+        };
+
+        const usTouch = {
+            TTCOnly: false,
+            HTOnly: true,
+            withTTC: false
+        };
+
+        const deutchTouch = {
+            TTCOnly: true,
+            HTOnly: false,
+            withTTC: true
+        };
+
+        const showTaxes = {
+            CA: usTouch,
+            WE: usTouch,
+            WS: usTouch,
+            QC: usTouch,
+            DE: deutchTouch,
+            FI: deutchTouch,
+            SN: deutchTouch,
+            CZ: frenchTouch,
+            ES: frenchTouch,
+            FR: frenchTouch,
+            GB: frenchTouch,
+            IE: frenchTouch,
+            IT: frenchTouch,
+            LT: frenchTouch,
+            MA: frenchTouch,
+            NL: frenchTouch,
+            PL: frenchTouch,
+            PT: frenchTouch,
+            TN: frenchTouch
+        };
+
+        function format (price, country, frequency) {
+            country = country || "FR";
+
+            const taxes = showTaxes[country];
+            if (price.withTax.value !== 0) {
+                if (taxes.TTCOnly) {
+                    return `<span class="bold">${price.withTax.text}</span>`;
```

Comment:
Why do you handle the HTML here ? You could handle it in HTML instead

This looks more like a `component` than a `filter`

Reply:
I took it from the web universe and cleaned up the code. It'll surely be moved to a "component" or whatever.